content/en/functions/dateformat.md

```diff
@@ -17,7 +17,7 @@ relatedfuncs: [Format,now,Unix,time]
 deprecated: false
 ---
 
-`dateFormat` converts the textual representation of the `datetime` into the specified format or returns it as a Go `time.Time` type value. These are formatted with the layout string.
+`dateFormat` converts an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp string `INPUT` into the format specified by the `LAYOUT` string.
```

Comment:
You are right about the return value (a string), but INPUT does not need to be ISO 8601 -- we support a bunch of different date/time formats, so if it looks like a date, it should work.

Reply:
Perfect, thanks. Should be fine now.